tests/load-tests/ci-scripts/stage/collect-results.sh

```diff
@@ -4,56 +4,114 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
-# construct $PROMETHEUS_HOST by extracting BASE_URL from $STAGE_MEMBER_CLUSTER
-BASE_URL=$(echo $STAGE_MEMBER_CLUSTER | grep -oP 'https://api\.\K[^:]+')  
-PROMETHEUS_HOST="thanos-querier-openshift-monitoring.apps.$BASE_URL"
-
+source "$( dirname $0 )/../utils.sh"
 
-# Login to the stage member cluster with the OCP_PROMETHEUS_TOKEN credentials
-TOKEN=${OCP_PROMETHEUS_TOKEN}
-oc login --token="$TOKEN" --server="$STAGE_MEMBER_CLUSTER"
+echo "[$(date --utc -Ins)] Collecting load test results"
 
+# Setup directories
 ARTIFACT_DIR=${ARTIFACT_DIR:-.artifacts}
+CONCURRENCY="${1:-1}"
 mkdir -p ${ARTIFACT_DIR}
-pushd "${1:-./tests/load-tests}"
+pushd "${2:-./tests/load-tests}"
 
-echo "Collecting load test results"
-cp -vf *.log "${ARTIFACT_DIR}"
-cp -vf load-tests.json "${ARTIFACT_DIR}"
+# Construct $PROMETHEUS_HOST by extracting BASE_URL from $STAGE_MEMBER_CLUSTER
+BASE_URL=$(echo $STAGE_MEMBER_CLUSTER | grep -oP 'https://api\.\K[^:]+')
+PROMETHEUS_HOST="thanos-querier-openshift-monitoring.apps.$BASE_URL"
+TOKEN=${OCP_PROMETHEUS_TOKEN}
 
-monitoring_collection_log=$ARTIFACT_DIR/monitoring-collection.log
-monitoring_collection_data=$ARTIFACT_DIR/load-tests.json
+echo "[$(date --utc -Ins)] Collecting artifacts"
+find . -maxdepth 1 -type f -name '*.log' -exec cp -vf {} "${ARTIFACT_DIR}" \;
+find . -maxdepth 1 -type f -name '*.csv' -exec cp -vf {} "${ARTIFACT_DIR}" \;
+find . -maxdepth 1 -type f -name 'load-test-options.json' -exec cp -vf {} "${ARTIFACT_DIR}" \;
+find . -maxdepth 1 -type d -name 'collected-data' -exec cp -r {} "${ARTIFACT_DIR}" \;
 
-## Monitoring data
-echo "Setting up tool to collect monitoring data..."
+echo "[$(date --utc -Ins)] Setting up Python venv"
+{
 python3 -m venv venv
 set +u
 source venv/bin/activate
 set -u
 python3 -m pip install -U pip
 python3 -m pip install -e "git+https://github.com/redhat-performance/opl.git#egg=opl-rhcloud-perf-team-core&subdirectory=core"
+python3 -m pip install tabulate
+python3 -m pip install matplotlib
```

Comment:
```suggestion
python3 -m pip install matplotlib
python3 -m pip install 'tenacity<8.4.0'
```

Recent changes to tenacity module cause failues - see https://github.com/jd/tenacity/issues/471

Reply:
I hope I have removed Tenacity dependency on OPL side. We have been only using one function from it anyway.